main/contracts/contracts.ts

```diff
@@ -61,6 +61,19 @@ export interface IMessage<T = any> {
  * Represents a messagebroker and provides access to the core features which includes publishing/subscribing to messages and RSVP.
  */
 export interface IMessageBroker<T> {
+    /**
+     * A unique identifier for this instance of the MessageBroker. This is useful for identifying an instance within a tree of scopes.
+     */
+    readonly name: string;
```

Comment:
I've introduced the name property on the messagebroker instance itself to keep track. The root broker has the name 'root'